client/RelatedItems&Comparison/OutfitList.jsx

```diff
@@ -89,7 +91,7 @@ class OutfitList extends React.Component {
     const {selectProductId, selectAnotherProduct, addNewOutfit, deleteOutfit, userOutfits} = this.props;
     return (
       <div className="outfitList">
-        <button type="button" className="leftArrow" style={{visibility: leftArrowVisibility}} onClick={this.moveToPrevCard}>&lt;</button>
+        <img src='./img/left-arrow-button.png' alt="left-arrow" className="leftArrow" style={{visibility: leftArrowVisibility}} onClick={this.moveToPrevCard} />
```

Comment:
If the linter is giving you issues, perhaps you need to assign role, tabIndex, and onKeyPress attributes to your images? Not sure what the linter issues are saying, so perhaps not.